conf/common.cfg

```diff
@@ -0,0 +1,27 @@
+[db]
+# type of database
+backend_grid = django.db.backends.mysql
+backend_cloud = django.db.backends.mysql
+
+# host with database
```

Comment:
Most of these comments are a bit unnecessary. A single description line for each section `[db cloud]` and `[db grid]` section should be enough. The only one that might need its own comment is the `backend` option to explain that this refers to the Django db backend.

Reply:
I tried to copy the content and maintain consistency.  But, It does make sense to me to NOT have comments that are bit unnecessary, Adrian. d50a671486dcf92392f0e84dbe1874abf75bdc71